### DISCUSSION TITLE
How to migrate from serenity-js/protractor to serenity-js/playwright?

### DISCUSSION THREAD

**Discussion by karthikcamarakanti-eaton** [serenity-js/serenity-js#2474] [How to... ?] [answered]
@jan-molak  I was trying to migrate from serenity/cucumber/protractor to serenity/Playwright/cucumber but facing some issues to setup the configuration files and capture logs in playwright like I do in protractor

**Comment by jan-molak** [comment, marked as answer]
Hi @karthikcamarakanti-eaton, did you have a look at the [Protractor migration guide](https://serenity-js.org/handbook/getting-started/serenity-js-with-protractor/#migrating-from-protractor-to-serenityjs)?

In the last step, you'd need to follow the [Serenity/JS+Cucumber+Playwright](https://github.com/serenity-js/serenity-js-cucumber-playwright-template) implementation instead of WebdriverIO, as described in the guide.
